libraries/standard/common/include/iot_linear_containers.h

```diff
@@ -496,6 +496,11 @@ static inline void IotListDouble_Remove( IotLink_t * const pLink )
     IotContainers_Assert( pLink != NULL );
 
     /* This function must be called on a linked element. */
+    /* Coverity finds a false positive in the below assert.
+     * As this is only an assert, the sizeof operator should
+     * not be called. Regardless, as IotLink_IsLinked does not
+     * mutate any data, there is no side effect in calling it. */
+    /* coverity[misra_c_2012_rule_13_6_violation] */
```

Comment:
Does this violation still come up in the release type build?

Reply:
No, it doesn't. It also doesn't come up if asserts are disabled.

Reply:
Let's remove the annotation then and run Coverity with asserts disabled.

Reply:
Reverted